iron-menu-behavior.js

```diff
@@ -138,6 +139,8 @@ export const IronMenuBehaviorImpl = {
 
     this.items.forEach(function(item) {
       item.setAttribute('tabindex', item === selectedItem ? '0' : '-1');
+      item.setAttribute(
+          'aria-selected', item === selectedItem ? 'true' : 'false');
```

Comment:
This looks like it will only set `aria-selected` to `true` for the first selected item if there's more than one. (`tabindex` intentionally behaves this way so that the menu only appears as one item in the sequential focus navigation order.)

Reply:
Good point. Changed it so that it's set on all selected items.